modules/oxide/package.json

```diff
@@ -36,6 +37,16 @@
     "prismjs": "^1.27.0"
   },
   "devDependencies": {
-    "gulp": "^4.0.2"
+    "connect": "^3.7.0",
+    "connect-livereload": "^0.6.1",
+    "grunt": "^1.6.1",
+    "grunt-contrib-clean": "^2.0.1",
+    "grunt-contrib-cssmin": "^5.0.0",
+    "grunt-contrib-less": "^3.0.0",
+    "grunt-stylelint": "^0.20.1",
+    "less": "^4.2.0",
+    "stylelint": "^16.6.1",
+    "stylelint-config-recommended-less": "^3.0.1",
+    "stylelint-order": "^6.0.4"
```

Comment:
```suggestion
```
Similar to the icons package, these don't need to be specified twice

(nor does grunt, please remove that too)